frontend/src/components/UsernameMenu.tsx

```diff
@@ -0,0 +1,42 @@
+import { useAuth0 } from '@auth0/auth0-react';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from '@radix-ui/react-dropdown-menu';
+import { Separator } from '@radix-ui/react-separator';
+import { CircleUserRound } from 'lucide-react';
+import { Link } from 'react-router-dom';
+import { Button } from './ui/button';
+
+const UsernameMenu: React.FC = () => {
+  const { user, logout } = useAuth0();
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger className="flex items-center gap-2 px-3 font-bold hover:text-orange-500">
+        <CircleUserRound className="text-orange-500" />
+        {user?.email}
+      </DropdownMenuTrigger>
+      <DropdownMenuContent>
+        <DropdownMenuItem asChild>
+          <Link to="/user-profile" className="font-bold hover:text-orange-500">
+            User Profile
+          </Link>
+        </DropdownMenuItem>
+        <Separator />
+        <DropdownMenuItem asChild>
+          <Button
+            onClick={() => logout()}
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/13#issuecomment-2362911772>#bb8a5a</a><span> - 09/20/2024, 06:23 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The logout function is called without any options, which may not properly clear all Auth0 session data.
</div>
<div id="fix">
<b>Fix:</b> Update the logout function call to include the necessary options for a complete logout process, including clearing all Auth0 session data.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -11,7 +11,7 @@
  import { Button } from './ui/button';
 
  const UsernameMenu: React.FC = () => {
 -  const { user, logout } = useAuth0();
 +  const { user, logout } = useAuth0();
 
    return (
      <DropdownMenu>
 @@ -28,7 +28,12 @@
          <Separator />
          <DropdownMenuItem asChild>
            <Button
 -            onClick={() => logout()}
 +            onClick={() => logout({
 +              logoutParams: {
 +                returnTo: window.location.origin,
 +              },
 +              openUrl: true,
 +            })}
              className="flex-1 font-bold bg-orange-500"
            >
              Log Out
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged